src/api/namespaces/neftyquest/format.ts

```diff
@@ -0,0 +1,37 @@
+export function formatQuest(row: any): any {
+    if (!row) {
```

Comment:
when is there an empty row sent? is it something to warn about or expected behaviour?

Reply:
I removed it